docs/common_issues.rst

```diff
@@ -5,7 +5,7 @@ Bulk Creating and Queryset Updating
 -----------------------------------
 ``django-simple-history`` functions by saving history using a ``post_save`` signal
 every time that an object with history is saved. However, for certain bulk
-operations, such as bulk_create_ and `queryset updates <https://docs.djangoproject.com/en/2.0/ref/models/querysets/#update>`_,
+operations, such as bulk_create_, `bulk_update, and `queryset updates <https://docs.djangoproject.com/en/2.2/ref/models/querysets/#update>`_,
```

Comment:
the the link `bulk_update` should be:  https://docs.djangoproject.com/en/3.0/ref/models/querysets/#bulk-update

You can use the underscore notation like on bulk_create